UI/window-basic-main.cpp

```diff
@@ -7893,7 +7880,7 @@ void OBSBasic::ProgramViewContextMenuRequested(const QPoint &)
 	popup.exec(QCursor::pos());
 }
 
-void OBSBasic::PreviewDisabledMenu(const QPoint &pos)
+void OBSBasic::PreviewDisabledMenu(const QPoint &)
```

Comment:
Parameter can be removed

Reply:
Only works with auto-generated slots.

Reply:
See https://github.com/obsproject/obs-studio/pull/8167#discussion_r1118067680

Reply:
Fixed. Only seems to work for me when I convert it to an auto-generated slot.

Reply:
That's probably the better solution anyways.